src/config/mod.rs

```diff
@@ -1368,6 +1370,7 @@ impl Default for DbConfig {
             lockcf: LockCfConfig::default(),
             raftcf: RaftCfConfig::default(),
             titan: TitanDbConfig::default(),
+            track_and_verify_wals_in_manifest: false,
```

Comment:
more of a question, but given that [rocks recommends setting this in production](https://github.com/facebook/rocksdb/wiki/Track-WAL-in-MANIFEST), what are the implications of setting the default here to true?

Reply:
This is simply to keep the behavior same as before. But you are right, maybe it is better to make it default true. What's your opinion? @tonyxuqqi

Reply:
@v01dstar In existing release branch's CP, we should make default value false. In master branch, I think it can be true.

Reply:
Done